engine/src/test/java/io/camunda/zeebe/engine/processing/streamprocessor/SkipFailingEventsTest.java

```diff
@@ -165,7 +167,8 @@ public void processRecord(final TypedRecord<UnifiedRecordValue> record) {
                       throw new NullPointerException();
                     }
                   });
-        });
+        },
+        Optional.empty());
```

Comment:
:thought_balloon: 
Wondering whether we can add a method with this optional instead giving it always to the method here?